gravitee-apim-portal-webui-next/src/entities/subscription/subscription.ts

```diff
@@ -35,6 +35,11 @@ export interface SubscriptionData {
 export interface SubscriptionLinks {
   self?: string;
 }
+
 export interface SubscriptionMetadata {
   [key: string]: { [key: string]: object | string };
 }
+
+export const Subscriptions = {
```

Comment:
The subscription statuses will always match these values: https://github.com/gravitee-io/gravitee-api-management/blob/master/gravitee-apim-rest-api/gravitee-apim-rest-api-portal/gravitee-apim-rest-api-portal-rest/src/main/resources/portal-openapi.yaml#L4366

The objects exported from this file should match what we expect to receive from the back. It is then up to the renderer/`api-tab-subscriptions` component to format the information (aka turn it into capitalize or lowercase).

Reply:
Done